uiowahjmjohnsonselt2024/projectdirectory-selt_2024_team_009
# Pull Request: Dynamic page title for each page

## What does this PR do?: 
This pull request adds the page title functionality to the app so that user can see in which page the user is currently in. Moreover, page title also helps in Search Engine Optimization.

## Related Issues: 
Closes #83 

## Testing
1. **Setup:** Pull the branch and run the application locally.
2. **Verify title:**
   - Verify if the different page shows different page title.
   - Create new page without a page title and check if the default page title works.

## Checklist
